runtime/compiler/control/J9Options.cpp

```diff
@@ -1461,44 +1461,41 @@ J9::Options::fePreProcess(void * base)
 
          char *optionsString = NULL;
 
-         /* Get Pointer to entire -Xlp:codecache: options string */
-         GET_OPTION_OPTION(lpccIndex, ':', ':', &optionsString);
+         // Get Pointer to entire "-Xlp:codecache:" options string
+         GET_OPTION_OPTION(xlpCodeCacheIndex, ':', ':', &optionsString);
 
-         /* optionsString can not be NULL here, though it may point to null ('\0') character */
+         // optionsString can not be NULL here, though it may point to null ('\0') character
```

Comment:
BTW OMR and de-facto OpenJ9 coding standards do not allow to use `//` type of comments but recommend `/* */ instead. I know that JIT developers ignore this/use own traditionally, however just let you know

Reply:
Also I would not mix up functional change and massive change in comments format

Reply:
> I know that JIT developers ignore this/use own traditionally, however just let you know

That's true. There are many coding guidelines that the JIT seems to ignore. You must be suggesting to create an issue to fix this.

Reply:
> Also I would not mix up functional change and massive change in comments format

I didn't necessarily think that changing two dozen lines of comments in change-relevant code to keep file comment uniformity was inappropriate. I would understand if these were comments else where in the file, but I think we're splitting hairs here. 

Let me know if you would like to me to make any changes, otherwise let's resolve this.